asyncpg/connection.py

```diff
@@ -2235,6 +2236,10 @@ async def connect(dsn=None, *,
         or the value of the ``PGTARGETSESSIONATTRS`` environment variable,
         or ``"any"`` if neither is specified.
 
+    :param str krbsrvname:
```

Comment:
Mention the new parameter in a `.. versionadded::` block below please.

Reply:
Done.